packages/babel-preset-fbjs/plugins/inline-requires.js

```diff
@@ -39,6 +39,15 @@
 module.exports = babel => ({
   name: 'inline-requires',
   visitor: {
+    CallExpression(path, state) {
+      const node = path.node;
+      // Rename an existing require function to make sure any inserted `require` calls point to the
+      // global CommonJS require function.
+      const requireBinding = path.scope.getBinding('require');
+      if (requireBinding != null && requireBinding.scope.parent != null) {
+        path.scope.rename('require');
+      }
+    },
```

Comment:
This has the undesired side effect that you might be renaming `require` functions even if the file isn't using any `require` call. Plugins should aim to minimize changes to the input source. I would further argue that this change would already guarantee that you never have a `global` `require` function declared since all are renamed. 

I suggest that we move the renaming to the place where we perform the `inlining`. Before you inline a call, rename `require` in the current scope to `_require` (or `importDefault` to `_importDefault`, see my comment below`. and then inline the call.